packages/components/pages/f-account-info/src/store/accountInfo.module.js

```diff
@@ -28,6 +29,12 @@ export default {
             commit(UPDATE_CONSUMER_DETAILS, { details, address });
         },
 
+        async saveConsumerDetails ({ state }, { api, authToken }) {
```

Comment:
PR main change here

Reply:
I might have missed some context but does this belong here, it doesn't seem to really save anything to the store from what i can see and looks like just an api call which should probably go in a service instead, again I could have missed something obvious here.

Reply:
The edit updates the store as each field is changed but the submit button then the saves/commits them to the api